ipaserver/install/installutils.py

```diff
@@ -154,7 +154,7 @@ def verify_fqdn(host_name, no_host_dns=False, local_hostname=True):
 
     try:
         # make sure that the host name meets the requirements in ipalib
-        validate_hostname(host_name)
+        validate_hostname(host_name, maxlen=64)
```

Comment:
I was afraid this would be confusing. 
Because the same validator(s) are used for both DNS and for some hostnames I did this whole maxlen juggle. Then there is the fact that host-add should be flexible, hence the configuration option.
The biggest leap I'm taking is that anything related to installation, I'm assuming must be on Linux, needs to be restricted to 64 max hostnames, regardless of config (which may or may not actually be available or easy to get at the time or place we need it).
I guess I should set the absolute minimum allowed value for maxhostname to 64 which might clarify some of this, would that help? 
I also considered creating a constant MAXHOSTNAMELEN and using that instead of 64 so it's less a "magic" number but I was afraid it would add to the confusion.
It occurs to me a doc block in the host plugin would probably be handy too.

Reply:
- if the configurable value applies only to ipa host-add, then I think you could simply modify the doc string of `ipa config-mod --maxhostname` so that it explains to which operations it applies.
- +1 for setting the absolute minimum allowed value for maxhostname to 64

Reply:
I also want it to apply with the --hostname option so the ipa-*-install fails more gracefully.